app/models/explainer.rb

```diff
@@ -106,7 +108,7 @@ def self.search_by_similarity(text, language, team_id)
         language: language
       }
     }
-    response = Bot::Alegre.request('post', '/text/similarity/search/', params)
+    response = Bot::Alegre.get_sync_raw_params(params, "text")
```

Comment:
I see this line is the same as line 85 above... but line 85 is sending content to be indexed in Alegre, while this line is searching for similar content from Alegre... so why they are the same?

Reply:
We don't have separate functionality for sending items to index vs getting search results - we do it all in one step either with async or sync. We can.... create new functionality to break that up?